README.md

```diff
@@ -253,7 +253,7 @@ from mobly.controllerse import android_device
 class HelloWorldTest(base_test.BaseTestClass):
 
   def setup_class(self):
-    self.ads = self.register_controller(android_device)
+    self.ads = self.register_controller(android_device, min_number=2)
```

Comment:
I don't think it's self explanatory, especially for people who don't really know what register_controller does. It seems obvious to you because you implemented :)